ikarus/io/resultfunction.hh

```diff
@@ -45,7 +45,7 @@ namespace Impl {
  * \tparam UserFunction Type of the user-defined function for custom result evaluation (default is
 DefaultUserFunction)
  */
-template <typename FE, ResultType resType, typename UserFunction = Impl::DefaultUserFunction>
+template <typename FE, template <typename, int, int> class resType, typename UserFunction = Impl::DefaultUserFunction>
```

Comment:
This should be named RT not resType

Reply:
Ah wait, it doenst need to be fixed at compile time